packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -166,6 +172,7 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
           )
         }>
         <div
+          style={maxRows != null && isCollapsed ? {maxHeight: tagState.maxHeight, overflow: 'hidden'} : undefined}
```

Comment:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/309540/218895773-953d3fbb-8867-47a4-8357-5bac7b920098.png">
I went to the "with-label-description-contextual-help" and set the maxRows to one. I narrow the container until the button disappears. I'm not able to reproduce this on main.

When I remove the `overflow: hidden` the button shows up where the `description` text is.

Reply:
Previously, it would always show at least one tag, but I've updated the logic to handle this edge case.